netdata-installer.sh

```diff
@@ -456,11 +456,17 @@ bundle_libmosquitto() {
     return 0
   fi
 
+  if [ "$(uname)" != "Linux" ]; then
+    echo >&2 " Sorry NetData with custom libmosquitto is unable on $(uname) at this time!"
+    echo >&2 " Please contact NetData suppoort! https://github.com/netdata/netdata/issues/new"
+    return 0
```

Comment:
```suggestion
    return 1
```

You are returning 1 in the eBPF check, should this also return 1?

Reply:
No the way @Ferroin wrote this it needs to `return 0` :)